README.adoc

```diff
@@ -138,10 +146,10 @@ include::finish/src/main/java/io/openliberty/guides/inventory/InventoryResource.
 
 The inventory resource is a RESTful service that is served at the `inventory/systems` endpoint. 
 
-Add the [hotspot=RequestScoped file=1]`@RequestScoped` annotation on the class to indicate that this bean is
-to be initialized once for every request. In other words, the bean is instantiated when the request
-is received and destroyed when a response is sent back to the client. While this bean can
-also be application-scoped, request scope is short-lived and is therefore ideal for HTTP requests.
+Add the [hotspot=ApplicationScoped file=1]`@ApplicationScoped` annotation on the class to indicate that this bean is
+to be initialized once for the duration of the application, where the state of the bean is shared between all the requests.
```

Comment:
Change
"Add the [hotspot=ApplicationScoped file=1]`@ApplicationScoped` annotation on the class to indicate that this bean is
to be initialized once for the duration of the application, where the state of the bean is shared between all the requests."
To
"Annotating a class with the [hotspot=ApplicationScoped file=1]`@ApplicationScoped` annotation indicates that the bean is initialized once and is shared between all requests while the application runs."